sharding/sharding.go

```diff
@@ -189,12 +199,14 @@ func (r *ShardingFerry) Run() {
 	r.Ferry.FlushBinlogAndStopStreaming()
 	copyWG.Wait()
 
+	// Joined tables cannot be easily monitored for changes in the binlog stream
+	// so we copy and verify them once more during the cutover phase
```

Comment:
This wording is a bit confusing. the "once more" is about "verify", not "copy", right? we don't actually copy them twice, but that's what this makes it sound like.

Reply:
We do actually. Before the cutover phase, the DataIterator batch copies the table. The "delta-copy" here ensures that we've copied over any new rows that have appeared since.